epicsdbbuilder/mydbstatic.py

```diff
@@ -86,10 +86,15 @@ def ImportFunctions(epics_base, host_arch):
 _libdb = None
 
 
-def GetDbFunction(name):
+def GetDbFunction(name, restype, argtypes, errcheck=None):
```

Comment:
These could be optional: I think `ctypes` has vaguely sensible defaults, and is it possible that argtypes have already been set elsewhere?  I'm not sure what happens if you ask for the same function twice from a library: do you get the same object?  If so, it's quite possible these may have already been set...